huggingface/base/inference/artifacts/mms-entrypoint.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import shlex
+import subprocess
+import sys
+
+if sys.argv[1] == 'serve':
+    from sagemaker_mxnet_serving_container import serving
```

Comment:
This should also be replaced by the `sagemaker_huggingface_inference_toolkit` later.